glue/plugins/dendro_viewer/qt/tests/data/dendro_v0.glu

```diff
@@ -234,6 +236,7 @@
       "_type": "glue.core.visual.VisualAttributes",
       "alpha": 0.78,
       "color": "#404040",
+      "preferred_cmap": null,
```

Comment:
In general, we should avoid modifying any of the .glu files in the test suite, since these double up as backward-compatibility tests. We should make the deserializer for visual robust to missing ``preferred_cmap``

Reply:
Understood. Will revert the changes now...

Reply:
Done, rebased and pushed